deps/ccommon/cmake/CMakeCargo.cmake

```diff
@@ -300,7 +325,7 @@ function(cargo_build)
         add_custom_command(
             # Dummy file to ensure that the command always runs
             OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/DOES_NOT_EXIST
-            COMMAND ${CARGO_ENV_COMMAND} ${CARGO_EXECUTABLE} build ${CRATE_ARGS}
+            COMMAND ${CARGO_ENV_COMMAND} cargo build ${CRATE_ARGS}
```

Comment:
What's the reason for this change?

Reply:
`CARGO_EXECUTABLE` was originally exported by `FindRust.cmake`. Since that's been removed in this PR the variable is no longer set so instead it just uses cargo directly.